src/EventTooltip.js

```diff
@@ -114,11 +116,9 @@ function formatComponentStack(componentStack: string): string {
 }
 
 const TooltipFlamechartNode = ({
```

Comment:
Not part of this diff, but happened to notice that the `COLORS.TOOLTIP_BG` and `COLORS.TOOLTIP` constants are not defined on `COLORS` according to flow (and my visual inspection), which may need to be addressed.

Reply:
Filed a more general issue to fix Flow types at #38. I've been ignoring the errors above as they haven't seemed to break anything, but we definitely will want to look at this again once the larger tasks are done.